### DISCUSSION TITLE
How to run on linux ?

### DISCUSSION THREAD

**Discussion by Karduin** [Durtur/Dungeoneer#94] [Q&A] [answered]
sorry for this stupid question, how do I run application under linux?

**Comment by ironjan** [comment, marked as answer]
Since there are no pre-built packages, you need to follow https://github.com/Durtur/Dungeoneer#contributing and build the application yourself.  ;)

> To get started you need to install nodejs, npm and yarn. Clone the project and run yarn install && yarn start.

Instead of cloning you can also download the source code zip or tar.gz of one of the releases and unpack it. Then navigate into the project folder with your command line and run the commands `yarn install && yarn start`.

----

One important addition: some Linux distributions come with quite old development tools which may cause some issues. After installing the requirements, you should be able to check the versions on your system with
```bash
npm --version
yarn --version
node --version
``` 
and then compare them to the current releases.

**Comment by Durtur** [comment]
Yes basically you need to build it yourself. I'm sorry for the lack of support but since I don't use Linux and the electron-builder package has no support for building Linux on a windows machine  I've not been including the linux builds in recent releases.
